src/components/InstanceBrowseFilters/InstanceFilterBrowse.js

```diff
@@ -72,6 +81,8 @@ const InstanceFiltersBrowse = props => {
 
   return (
     <AccordionSet accordionStatus={accordions} onToggle={onToggleSection}>
+      {
+      browseType === 'callNumbers' && (
```

Comment:
Please compare with `browseModeOptions` constant values here

Reply:
Thanks, I fixed it